operators/src/util/raster_stream_to_png.rs

```diff
@@ -60,9 +76,10 @@ where
                 Err(error) => futures::future::err(error),
             }
         })
-        .await?;
+        .await;
 
-    Ok(output_tile.grid_array.to_png(width, height, &colorizer)?)
+    info!("output tile:  {}", output_tile.is_ok()); // TODO: remove
```

Comment:
Then remove?

Reply:
This is an experimental PR and I left this outputs on purpose for you to see what happens.